README.md

```diff
@@ -9,26 +9,67 @@ A practical functional library for Javascript programmers.
 [![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/ramda/ramda?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 
-Goals
------
+Why Ramda?
+----------
 
 <img src="http://ramda.jcphillipps.com/logo/ramdaFilled_200x235.png" 
-     width="200" height="235" align="left" hspace="12" />
-Using this library should feel as much like using Javascript as 
-possible.  Of course it's functional Javascript, but we're not 
-introducing lambda expressions in strings, we're not borrowing consed 
+     width="200" height="235" align="right" hspace="12" />
+
+Have you ever dreamed about snapping code together just like lego bricks ?
+You don't need to learn a functional programming language to reap the
+rewards of simpler code. Ramda makes it simple for you to build complex
+logic through functional composition (don't worry this only means sticking
+lego bricks together).
+
+Because of design decisions that you don't need to worry about either, Ramda
+can get your job done more elegantly, more flexible and with less code.
+
+Compare
+-------
+
+Let's compare the typical `LoDash` or `Underscore` chain style
+
+```js
+     var phonesSortedByName = function(users){
+          return _.chain(users)
+               .sortBy(function(user){
+                    return user.name;
+               })
+               .map(function(user){ return user.phone })
+               .value();
+     }
+```
+with Ramda's *list comes last* (Lego®) style
+
+```js
+     var phonesSortedByName = R.compose(
+               R.map(R.pluck('phone')),
+               R.sortBy(R.pluck('name'))
+     );
+```
+
+While LoDash or Underscore are great libraries, they sometimes introduce
+complexities in your code which Ramda avoids with a simpler design.
```

Comment:
Yes, we're having enough problems with people thinking that ours is meant to be an `Underscore/Lo-Dash` replacement, and enough problems from the author of one of those seeming to see us as a competitor.  Let's not throw down the gauntlet here.

If we want to do this sort of comparison, it would be better to compare with native code.  Blog posts and such can do the library comparison.  (And the ReadMe could link to those if they're important enough.)